migration/sql-files/005-add-search-index.sql

```diff
@@ -0,0 +1,2 @@
+CREATE INDEX IF NOT EXISTS fields_title ON work_items USING GIN (to_tsvector('english', fields->>'system.title'));
```

Comment:
I have created https://github.com/almighty/almighty-core/issues/422 for adding general indexing for fields